.github/workflows/tests.yaml

```diff
@@ -0,0 +1,107 @@
+name: Tests
+on: [push, pull_request_target]
+
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    if: >
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
Can you elaborate the use of having the `github.event.pull_request.head.repo.full_name != github.repository` here @laurent-yoti . I tried to find out but seemed they both give the repository name `getyoti/yoti-node-sdk-sandbox`. 
Since we have `!=` here, would like to know about the use of this condition.

Reply:
Hum, good spot...
I have no real answers...
I have been thinking that the only way for that not to be true would be if there is a fork trying to do a PR...
...and apparently my guess was correct as there is some comment in that file saying:

```# always run on push events
 #    # only run on pull_request_target event when pull request pulls from fork repository```

Reply:
Ah okay, make sense now. Thank you Laurent.